config/index.js

```diff
@@ -58,4 +104,4 @@ function getHashiConfig(configPath) {
 	}
 }
 
-module.exports = { loadHashiConfigForEnvironment, getHashiConfig }
+module.exports = { loadHashiConfigForEnvironment }
```

Comment:
People should be using `loadHashiConfigForEnvironment` as it already has sane defaults, `getHashiConfig` needs _you_ to set the file loading and defaults so we should only be using it internally within this file.